src/com/facebook/buck/swift/SwiftLibraryDescription.java

```diff
@@ -287,8 +292,15 @@ public BuildRule createBuildRule(
           cxxPlatform.getCpp().resolve(graphBuilder, buildTarget.getTargetConfiguration());
 
       BuildTarget buildTargetCopy = buildTarget;
+
+      ImmutableSortedSet.Builder<FrameworkPath> builder = ImmutableSortedSet.naturalOrder();
```

Comment:
This is duplicated code that should be refactored into a method if we want to move forward with this design.
I'm not doing it now because I prefer to adopt a different design.